config/prod/iatlas-db.yaml

```diff
@@ -10,5 +10,5 @@ parameters:
   KmsKey: !stack_output_external iatlas-kms-prod::KmsKey
   DbSecretPrefix: 'prod_'
   DbCredsName: 'iatlas_db_creds_encrypted_prod'
-  DBInstanceClass: 'db.t3.large'
+  DBInstanceClass: 'db.m6g.large'
```

Comment:
Should we not try staging only first?

Reply:
Yes, let's push to staging first and it will deploy to `staging`. If this is working, then we can push to `prod` to deploy to production. Making the changes in both configs so we don't need to deploy the same thing to staging twice.